### ISSUE TITLE
Support for Vaddio AV Bridge Matrix Pro.

### ISSUE THREAD

**Issue by dara-walsh** [bitfocus/companion-module-requests#279]
Hi, Im trying to control a Vaddio AV Bridge Matrix Pro with companion.
I can control it through Rs232 using coolterm but cant get streamdeck to connect to coolterm or direct to device.

Im a novice at this kind of stuff

Any help or direction would be appreciated 

D

**Comment by dara-walsh** [comment]
See page 51 for Serial Command API

https://res.cloudinary.com/avd/image/upload/v132443271/Resources/Vaddio/AV%20to%20USB%20Bridges%20and%20Encoders/Operation/411-0020-30-rev-d-av-bridge-matrix-pro-integrators-complete-guide.pdf

**Comment by josephdadams** [comment]
Companion can't connect via a local rs232/serial port, but the device supports telnet connections, so it should be possible to use that over the network.

**Comment by dara-walsh** [comment]
Yes Ive been looking into the telnet control and it look fairly simple but i would not know how to create a companion module.
I'm hoping to control 
1. program output (1-4)
2. Stream output (1-4)
3. PIP
4. Maybe camera adjustments Zoom and PTZ (1-4)
5. And camera presets

If anyone could help it would be great.

**Comment by dara-walsh** [comment]
Here is a view of the web interface
![Screenshot 2020-09-12 at 23 19 02](https://user-images.githubusercontent.com/59572459/93006078-c4ba8100-f54f-11ea-9de2-e431844c4848.png)

**Comment by dara-walsh** [comment]
I will commit to being a tester and help out any way i can
